frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -294,11 +295,13 @@ const LogsTabForPodName: React.FC<{ podName: string; isFailedPod: boolean }> = (
                     </ToolbarItem>
                     <ToolbarItem spacer={{ default: 'spacerNone' }}>
                       <Dropdown
+                        data-test-id="logs-kebab-toggle"
```

Comment:
Change this to `data-testid` for consistency, and also change its value because it is used below on line 304.

Reply:
removed this, coz I had added this twice. my bad.